lib/livebook/hub/fly.ex

```diff
@@ -0,0 +1,132 @@
+defmodule Livebook.Hub.Fly do
+  @moduledoc false
+  defstruct id: nil,
+            name: nil,
+            deployed: nil,
+            hostname: nil,
+            organization: nil,
+            image_details: nil,
+            state: nil,
+            token: nil
+
+  alias Livebook.Utils.HTTP
+
+  @doc false
```

Comment:
Since we test it and it returns the `%Fly{}` struct, it seems internally public (the module is doc false anyway)

```suggestion
```